etc/security/mh_default_org.xml

```diff
@@ -249,7 +249,8 @@
     <sec:intercept-url pattern="/export/**" method="GET" access="ROLE_ANONYMOUS" />
 
     <!-- Enable anonymous access to the annotation and the series endpoints -->
-    <sec:intercept-url pattern="/series/**" method="GET" access="ROLE_ANONYMOUS" />
+    <sec:intercept-url pattern="/series/**" method="GET" access="ROLE_ANONYMOUS, ROLE_CAPTURE_AGENT" />
+    <sec:intercept-url pattern="/assets/**" method="GET" access="ROLE_ADMIN, ROLE_CAPTURE_AGENT" />
```

Comment:
Do we explicitly need to add `ROLE_ADMIN`? Shouldn't that always work?
Why does a capture agent need read access to assets?

Reply:
Good catch. The `assets` rule was just for debugging purposes. I'll remove that.